index.js

```diff
@@ -300,9 +304,6 @@ Router.prototype.find = function (method, path) {
       path = path.slice(i)
       continue
     }
-
-    // route not found
-    if (len !== prefixLen) return null
```

Comment:
are we sure about this change?

Reply:
Yes, that code was forgotten there, its position is [here](https://github.com/delvedor/find-my-way/blob/e765c5b0df43049e03bdaf3b989fce0399713ab9/index.js#L245) :D